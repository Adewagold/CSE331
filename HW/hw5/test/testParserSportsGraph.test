# Test the parser on sports data to see 
# if the graph is built correctly

# Parse through the data and build the graph
LoadGraph sports sports.tsv

# List nodes and edges to see if the graph 
# is built correctly
ListNodes sports
ListChildren sports Michael
ListChildren sports Howard
ListChildren sports Jeff