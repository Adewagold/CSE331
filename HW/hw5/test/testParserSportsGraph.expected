# Test the parser on sports data to see 
# if the graph is built correctly

# Parse through the data and build the graph
loaded graph sports

# List nodes and edges to see if the graph 
# is built correctly
sports contains: Howard Jeff Michael
the children of Michael in sports are: Jeff(MLB) Jeff(NBA)
the children of Howard in sports are: Jeff(NFL) Jeff(NHL)
the children of Jeff in sports are: Howard(NFL) Howard(NHL) Michael(MLB) Michael(NBA)